pkgs/by-name/os/osc-cli/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "osc-cli";
+  version = "1.11.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "outscale";
+    repo = "osc-cli";
+    rev = "v${version}";
+    hash = "sha256-7WXy+1NHwFvYmyi5xGfWpq/mbVGJ3WkgP5WQd5pvcC0=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
```

Comment:
I am confused why this is required here

Reply:
`wheel` doesn’t seem to be needed, but without `setuptools` there is an error during build phase: `ModuleNotFoundError: No module named 'setuptools'`.

Reply:
I was specifically referring to wheel. That setuptools is required, is clear for me.